lighthouse-core/report/html/report-styles.css

```diff
@@ -541,8 +541,19 @@
   background-color: var(--color-hover);
 }
 
-/* Hide the expandable arrow icon, three ways: via the CSS Counter Styles spec, for webkit/blink browsers, hiding the polyfilled icon */
-/* https://github.com/javan/details-element-polyfill/blob/master/src/details-element-polyfill/polyfill.sass */
+/* We want to hide the browser's default arrow marker on summary elements. */
```

Comment:
```css
/* We want to hide the browser's default arrow marker on summary elements. Admittedly, it's complicated. */
.lh-audit-group > summary,
.lh-expandable-details > summary {
  /* Blink 89+ will hide the arrow when display is changed from (new) default of `list-item` to block.  https://chromestatus.com/feature/6730096436051968*/
  display: block;
}
/* Safari, Firefox (surprisingly), and Blink <=88 require using the -webkit-details-marker selector */
.lh-audit-group > summary::-webkit-details-marker,
.lh-expandable-details > summary::-webkit-details-marker {
  display: none;
}
/* When the details-element-polyfill is used, it adds an ugly arrow. Hide it. https://github.com/javan/details-element-polyfill/blob/main/src/styles.js */
.lh-audit-group > summary:before,
.lh-expandable-details > summary:before {
  display: none;
}
```

Reply:
I tweaked my comments a bit, but IMO it was already specifying why things were being done at an appropriate level of detail (new/old browsers, tale as old as time). The provided links provides more detail (like what version of blink the newer stuff is supported).

got rid of the `@supports` block too, the idea there was to eventually have chrome not emit a deprecation warning if it was smart and checked for these rules in a better way; but the deprecation is now removed, so whatever.

Reply:
I'm unclear why this has to be contentious.. 

> I tweaked my comments a bit, but IMO it was already specifying why things were being done at an appropriate level of detail

if that were the case, then you would have deleted the `list-style-type: none;` rule. it's totally unnecessary.